baselines/Gmail Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -696,7 +696,7 @@ To configure the settings for Spoofing and Authentication Protection:
 
 ## 8. User Email Uploads
 
-This section enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.
+This section deals with the feature that enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.
```

Comment:
One minor comment that might just be personal preference. "The feature" implies that this is the one and only way for users to do this. I'd prefer "a feature" instead of "the feature" as it's more open ended. 
```suggestion
This section addresses a feature that enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.
```